library/Freckle/App/Database.hs

```diff
@@ -20,8 +20,6 @@ module Freckle.App.Database
   , envPostgresPasswordSource
   ) where
 
-import Freckle.App.Prelude
-
```

Comment:
:art: This should stay up here. We separate a module's prelude import like this conventionally.

Reply:
We'd need to change restyled for that, is there an annotation I can use?

Reply:
Can you show be Restyled doing this?

If you combine the groups, it'll sort them (stylish-haskell), but (as far as I know) it will not combine the groups on you.

Reply:
You can see it in the PR restyled created. Though as you point out, this is likely because the blank line in between groups ended up with something on it.

Reply:
Yup. This was the commit Restyled (stylish-haskell) saw,

![](https://files.pbrisbin.com/screenshots/screenshot.3210036.png)

So you made it one big group, then it got sorted. If you put it back to separate, we'll be good.